_assets/js/utils/getOffsetParam.js

```diff
@@ -0,0 +1,10 @@
+// Get the current offset value:
+const getOffsetParam = () => {
+    const searchParams = new URLSearchParams(window.location.search);
+    const offset = searchParams.get("offset")
```

Comment:
Missing semi colon.

I wonder if you can run prettier to catch semi colons and whatnot?  check out the prettier rules we have set up in crt-portal

Reply:
This will happen once I implement the CRT build pipeline. That will have to wait until the DOJ is aware and signs off on the idea.